manageiq-operator/deploy/crds/manageiq.org_manageiqs_crd.yaml

```diff
@@ -51,6 +51,13 @@ spec:
             databaseVolumeCapacity:
               description: 'Database volume size (default: 15Gi)'
               type: string
+            disableApplicationLocalLogin:
```

Comment:
I'm finding the negatively named option with a default of false confusing.  Is it possible to make it `allowApplicationLocalLogin` with a default of `true` ?

Reply:
I was just matching what the setting in our database was, but I can definitely change it here if you think the translation won't be more confusing than the option name.

Reply:
I guess I don't like our decision at the database either :/

I don't know how hard it is to change CRDs later, which is why I'm thinking go positive now and we can change our application later, but I'm not sure...it's up to you.

Reply:
I think the CRD should be semver versioned (at least for OLM) so I think we can go to positive now.

Reply:
Hm, so when trying to change this the "default value" semantics are a bit difficult for booleans.

In Go the "zero value" for a bool is false so the case when someone doesn't set a value and when a user explicitly sets `false` look identical.

So if we want to keep this a bool I think the defalut value needs to remain `false`. Otherwise we could change it to a string, but that might be weird.

What do you think @Fryguy ?

Reply:
Well you know what I think of go LOLOLOLOL 🤣 

In go, is it possible to have the field be like a `Bool | Nil` union thing (i.e. Maybe or Option type from other langs)?  If so, then you have the possibility to differentiate "null" vs "false"

Reply:
So that might be an option, but I don't know how that would work for the CRD as it has to map to a type that the sdk knows about I think. Generally I'd rather not use a non-standard type in the CR if we can avoid it.

I've seen internal structs to this with pointers but I don't know how that would work for something that needs to be provided in a yaml file. For example: https://github.com/kubernetes/api/blob/a4f2d87006da76427d0d3d613f2ffec13d8e49d3/apps/v1/types.go#L274. I'll give this a shot.

Reply:
Ah ok, yeah that package I linked I think works in effectively the same way (i.e. use a pointer, and nil means not specified)

Reply:
The pointer conversion seems to have done the job.